Example/jaegerMediator/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "jaegermediator",
+  "version": "1.0.0",
+  "description": "A mediator service that takes in JSON spans, converts the data to Thrift and sends it to a Jaeger collector.",
+  "main": "app.js",
+  "scripts": {
+    "test": "node app.js",
+    "start": "node app.js"
+  },
+  "dependencies": {
+    "express": "^4.16.4",
+    "path": "^0.12.7",
+    "thriftrw": "^3.11.3"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
Please add the SPDX ID here for the license we use in the rest of the project. I believe it's `BSD-2-Clause`. Could you also add `"private": true` to prevent this package from being published?

Reply:
Just updated the licensing and made this package private